docs/source/cpp/compute.rst

```diff
@@ -286,7 +287,7 @@ an ``Invalid`` :class:`Status` when overflow is detected.
 +--------------------------+------------+--------------------+---------------------+
 | power_checked            | Binary     | Numeric            | Numeric             |
 +--------------------------+------------+--------------------+---------------------+
-| subtract                 | Binary     | Numeric            | Numeric (1)         |
+| subtract                 | Binary     | Numeric            | Numeric             |
```

Comment:
Was this intentional?

Reply:
Not intentional at all. This was sloppy on my part.